src/util/config.ts

```diff
@@ -36,17 +36,22 @@ async function fetchConfig(): Promise<Config> {
 				state[name] = parameter.Value;
 			}
 		}
+		console.log(`Configuration spreadsheet: ${state['spreadsheet.id']}`);
 		return state;
 	} else {
 		return Promise.resolve(state);
 	}
 }
 
 export async function getConfigItem(key: string): Promise<string> {
-	const config = await fetchConfig();
-	if (config[key]) {
-		return config[key];
-	} else {
-		throw new Error(`No config value for key: ${Path}`);
+	try {
+		const config = await fetchConfig();
+		if (config[key]) {
+			return config[key];
+		} else {
+			throw new Error(`No config value for key: ${Path}`);
```

Comment:
I think we should log which value couldn't be found, so

```suggestion
			throw new Error(`No config value for key: ${Path}/${key}`);
```